### ISSUE TITLE
[BUG] Illegal/Unsupported Characters

### ISSUE THREAD

**Issue by BigBagss** [Fermiz/Flow.Launcher.Snippets#3]
**tl;dr**: snippets containing these characters break/cause unintended behavior: `{, }, |, \, <, >, ^, &`
some of those show a error prompt (by using this snippet = `<>/;!@#$%^&*_-+|\`).

i am not an expert but i believe they break the plugin because they are unescaped/reserved as special syntax.

besides that, i think multi-line snippets are not supported.

# Examples:
when using the snippet (containing multiple lines, and illegal characters):
```
setTimeout(() => {
  //code
}, 2000);
```
(this multi line snippet was saved using a db editor)

- the copied clipboard item is empty 
- new files are created inside `%FlowLauncherPluginFolder%/Snippets/` with filenames containing parts of the snippet that includes those characters.

so using the example above, i see a new file called `{`, with the contents of: `setTimeout(() =`

**i believe this only happens to snippets containing multiple lines.**

when saving the snippet in a single line (`setTimeout(() => {//code}, 2000);`) i don't see a new file being created, but the copied clipboard item is still empty (probably because of the illegal characters `{, }`)

another example:
snippet: `<>/;!@#$%^&*_-+|\`

shows a Flow launcher error.

p.s: i used a db browser to save multi-lined snippets.
___

thank you for making this much needed plugin!

**Comment by haakonstorm** [comment]
Second this unfortunately. I tried adding snippets for some unicode characters like ctrl, alt, option, shift, but these all only return a question mark "?" when retrieved.

**Comment by Fermiz** [comment]
Sorry for replying late, let me do the enhancement. My idea is: instead of copy the `<snippets>` and then paste the `key:<snippets value>`, I would get all records from the clipboard and then we can choose to save the snippets directly instead of paste again.

**Comment by Fermiz** [comment]
The new version is merged https://github.com/Fermiz/Flow.Launcher.Snippets/pull/5

**Comment by Fermiz** [comment]
Hi @BigBagss, by the way, please check the latest readme for multiple line usage

**Comment by BigBagss** [comment]
awesome! after a quick test everything seems to work well.
btw, there is a small typo in the readme: "Save From **Clipbaord** for Multiple Line"

thanks for this plugin and the update!
